Documentation/AppA.tex

```diff
@@ -26,19 +26,25 @@ \section{Working Packages}
 \section{caption and subfig}
 This class file already formats captions for figures and tables according the requirements of the Graduate School.  As a result, the caption package, which allows you to manipulate how these elements appear, should not be used.
 
-The one exception to this is if you use the caption or subfig package (which depends on the caption package) to create multi-page floats.\footnote{If you need to create subfigures but don't need a figure to span multiple pages, use the subfigure package, as it won't conflict with this class file.} If you do, you'll find that the caption formats and the List of Figures/Tables do not typeset the way the Graduate School wants them to be typeset.  In particular, the separator between ``Figure \#.\#'' and the caption will be ``:\space'' instead of ``.\space'' and the word ``Figure'' or ``Table'' will not accompany the number in the respective list.  To avoid this you need to add the following code to your preamble:
+The one exception to this is if you use the caption or subfig package (which depends on the caption package) to create multi-page floats.\footnote{If you need to create subfigures but don't need a figure to span multiple pages, use the subfigure package, as it won't conflict with this class file.} If you do, you'll find that the caption formats and the List of Figures/Tables do not typeset the way the Graduate School wants them to be typeset.  In particular, the separator between ``Figure \#.\#'' and the caption will be ``:\space'' instead of ``.\space'' and the word ``Figure'' or ``Table'' will not accompany the number in the respective list.  To avoid this, the following code has been added to maine-thesis.cls:
+\begin{verbatim}
+\AfterPackage{subfig}{
+	\captionsetup{labelsep=period,listofformat=simple}
+	\renewcommand\p@figure{Figure\space}
+	\renewcommand\p@table{Table\space}
+}
+\end{verbatim}
```

Comment:
TMI.  The end user doesn't need to know what is happening behind the scenes, only the special actions they need to take when using these packages.